src/spec-configuration/httpOCIRegistry.ts

```diff
@@ -198,6 +220,34 @@ async function getCredential(params: CommonParams, ociRef: OCIRef | OCICollectio
 	return;
 }
 
+async function getCredentialFromHelper(params: CommonParams, registry: string, credHelperName: string) : Promise<{ base64EncodedCredential: string | undefined; refreshToken: string | undefined } | undefined>{
+	const { output } = params;
+
+	const { stdout } = await runCommandNoPty({
+		exec: plainExec(undefined),
+		cmd: 'docker-credential-'+credHelperName,
+		args: ['get'],
+		stdin: Buffer.from(registry, 'utf-8'),
+		output,
+	});
+	if (stdout.length === 0) {
+		return undefined;
+	}
+
+	const creds:CredentialHelperResult = jsonc.parse(stdout.toString());
```

Comment:
`jsonc.parse` throws an error if the string cannot be parsed. Should we catch that and return undefined here?